src/test/kotlin/no/nav/familie/ef/sak/iverksett/oppgaveterminbarn/ForberedOppgaverTerminbarnServiceTest.kt

```diff
@@ -0,0 +1,132 @@
+package no.nav.familie.ef.sak.iverksett.oppgaveterminbarn
+
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockk
+import io.mockk.mockkObject
+import io.mockk.runs
+import io.mockk.slot
+import io.mockk.verify
+import no.nav.familie.ef.sak.fagsak.FagsakService
+import no.nav.familie.ef.sak.iverksett.IverksettClient
+import no.nav.familie.ef.sak.opplysninger.personopplysninger.PersonService
+import no.nav.familie.ef.sak.opplysninger.personopplysninger.mapper.GrunnlagsdataMapper
+import no.nav.familie.ef.sak.opplysninger.personopplysninger.pdl.Fødsel
+import no.nav.familie.ef.sak.opplysninger.personopplysninger.pdl.PdlBarn
+import no.nav.familie.ef.sak.testutil.PdlTestdataHelper.fødsel
+import no.nav.familie.ef.sak.testutil.PdlTestdataHelper.pdlBarn
+import no.nav.familie.kontrakter.ef.iverksett.OppgaverForBarnDto
+import no.nav.familie.kontrakter.felles.ef.StønadType
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.time.LocalDate
+import java.util.UUID
+
+internal class ForberedOppgaverTerminbarnServiceTest {
+
+    private val personService: PersonService = mockk()
+    private val fagsakService: FagsakService = mockk()
+    private val terminbarnRepository: TerminbarnRepository = mockk()
+    private val iverksettClient: IverksettClient = mockk()
+    private val forberedOppgaverTerminbarnService =
+            ForberedOppgaverTerminbarnService(personService, fagsakService, terminbarnRepository, iverksettClient)
+    private val oppgaverForBarnSlot = slot<OppgaverForBarnDto>()
+    val fødsel: Fødsel = mockk()
+
+    @BeforeEach
+    fun init() {
+
+        every { fagsakService.hentAktivIdent(any()) } returns ""
+        every { iverksettClient.sendOppgaverForTerminBarn(any()) } just runs
+        every { personService.hentPersonMedBarn(any()).barn } returns mockk()
+        every { terminbarnRepository.insert(any()) } returns mockk()
+        every { fødsel.erUnder18År() } returns true
+        mockkObject(GrunnlagsdataMapper)
```

Comment:
`mockkObject` trenger `unmockkObject` etter at testene er kjørte, hvis ikke kan det skape krøll for andre tester då man mocket er statiskt objekt